runner/src/internal.rs

```diff
@@ -54,16 +54,25 @@ pub struct RunExamples {
     pub example_name: Option<String>,
     #[structopt(long, help = "only build the examples, do not run them")]
     pub build_only: bool,
+    #[structopt(flatten)]
```

Comment:
help text?

Reply:
This just means that the `BuildServer` struct is nested and flattened in this one, so the help text of those flags will appear here.